src/Mod/CAM/PathSimulator/AppGL/SimDisplay.cpp

```diff
@@ -508,11 +508,11 @@ void SimDisplay::MoveEye(float x, float z)
 {
     // Exponential calculate maxValue
     // https://forum.freecad.org/viewtopic.php?t=96939
-    const float arg1 = 124.938;
-    const float arg2 = 578.754;
-    const float arg3 = -20.7993;
-    float maxValueX = arg1 + arg2 * exp(arg3 * mEyeDistFactor);
-    float maxValueZ = maxValueX * 0.4;
+    const float arg1 = 124.938F;
+    const float arg2 = 578.754F;
+    const float arg3 = -20.7993F;
+    float maxValueX = arg1 + arg2 * std::expf(arg3 * mEyeDistFactor);
```

Comment:
Needs an include. I'm reverting this suggestion so we can merge it
```suggestion
    float maxValueX = arg1 + arg2 * exp(arg3 * mEyeDistFactor);
```